src/pug/blog/gyro_ratcheting.pug

```diff
@@ -0,0 +1,162 @@
+
+  p But when people started experimenting with this approach, due to the limitations of current gyros, it was obvious that some tradeoffs had to be made: a low sensitivity allowed for fine aiming, but made very tedious to make big turns (180°); on the contrary a higher sensitivity allowed for big turns, but made fine aiming very noisy and unreliable. This latest approach combined with a lot of post-processing (smoothing and acceleration) was the state-of-the-art solution for gyro ratcheting for some time.
+
+  p: em Pros
+  ul
+    li Single method of input, softer learning curve.
+    li It allows to build muscle memory, and to do long-travel flick shots.
+    li Mostly equivalent to using a mouse.
+    li It requires less complex movements on the thumb finger.
+  p: em Cons
+  ul
+    li Full range of motion requires a high sensitivity.
+    li High sensitivity requires a higher gyro performance for fine movements, either by smoothing, or other solutions (see later).
+    li It requires a higher range of motion in the hands and/or wrists.
+    li Requires to set the in-game sensitivity within an acceptable range (1:3 to 1:5).
+  hr
+
+  h3 Flick stick
+  p Flick stick is an evolution of the thumbstick+gyro parallel method, so there is still two parallel methods to control the camera. but now the thumbstick is used in a more clever way. Instead of the classic relative turn-rate control, the thumbstick is used as an absolute 1:1 turn control on the horizontal axis by rotating it around its center. So for example by doing a physical 180° turn on the thumbstick, the in-game camera also turns 180°.
+  p This is a clear improvement in many scenarios, but also comes with some tradeoffs: The much higher 1:1 sensitivity mapping makes the thumbstick more twitchy, less suited from fine aiming, so now the thumbstick is relegated to be a method to do fast #[em flicks] only, and the gyro takes a much more primary role.
+
+  p: em Pros
+  ul
+    li It does not require a high performance gyro sensor, so is usable with any controller with a basic IMU sensor.
+    li It allows for fast turns compared with traditional thumbstick-camera controls.
+  p: em Cons
+  ul
+    li Parallel input method, harder learning curve.
+    li Only suitable for a single axis (X-axis), so not good for general mouse emulation, or 6DOF games.
+    li It requires more complex finger/hands movements to perform the same actions.
+    li Requires a careful manual setup to set the in-game sensitivity to 1:1 ratio.
+    li Due to the super-high sensitivity / low-resolution required on the thumbstick, flick stick method is not very good for long distance flicks based on muscle memory (See table below).
+    //- This arguably forces the user to turn the stick approximately to the final destination, and then doing the last adjustments with the gyro. A two-step movement that by definition is not a flick shot.
+  hr
+
+  h3 Full scale resolution (long distance flick shots)
```

Comment:
Explanation of this section required. E.g., "The amount of movement, required to achieve the following in-game actions"